src/antd-components/Mentions.tsx

```
@@ -0,0 +1,41 @@
+
+type MentionsOptionProps = GetProp<MentionProps, "options">[number];
+
+const onChange = (value) => {
+  console.log("Change:", value);
+};
+
+const onSelect = (option) => {
+  console.log("select", option);
+};
+
+const Mentions = () => (
```

Biome

[error] Shouldn't redeclare 'Mentions'. Consider to delete it or rename it.

'Mentions' is defined here:


(lint/suspicious/noRedeclare)